vars/systemTestsCapsule.groovy

```diff
@@ -5,25 +5,26 @@ import org.jenkinsci.plugins.workflow.support.steps.build.RunWrapper
 
 void call(Map config = [:]) {
   Boolean ignoreFailure = config.ignoreFailure ? "${config.ignoreFailure}".toBoolean() : false
-  String systemTestsCapsuleJob = '/common/system-tests'
+  // TODO: update before merge
+  String systemTestsCapsuleJob = '/private/playgrounds/system-tests-one-repo'
```

Comment:
Is that meant to stay for now?

Reply:
This is temporary for testing with different branches. I will change it to the correct location right before the merge.